myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -200,17 +200,6 @@ def __init__(self,
         self.relief_range = relief_range
         self._populate_patches()
 
-    def flatten_agent_patch(self, qpos):
```

Comment:
Seems like throughout this file, we are overwriting newer commits. Be careful here.

Reply:
@P-Schumacher can we have your feedback on this ?